packages/block-editor/src/utils/transform-styles/transforms/test/url-rewrite.js

```diff
@@ -21,14 +21,6 @@ describe( 'URL rewrite', () => {
 		expect( output ).toMatchSnapshot();
 	} );
 
-	it( 'should not replace absolute paths', () => {
```

Comment:
Why did we remove this test?

Reply:
Yes, the test was removed because URLs that start with a slash (e.g. `/images/image.jpg`) are still relative URLs, just relative to the host and normalized (without `..`s). However, if there should be still an exception for host-relative URLs this can be added.

Reply:
So you're saying that rewriting them or not still results in the same URL. That's fine (though not rewriting seems safer), but let's keep the test.

Reply:
Alright, the test can be readded. Though the URL would be rewritten from `/something/` to `https://example.com/something/` because then it becomes an absolute URL.

Reply:
that's probably fine but since the spec is sometimes very large, it seems safer to just not rewrite it if possible.

Reply:
@youknow: OK, then I add a test for hose-relative URLs and just skip them.

Reply:
Code and test added. 👍